package.json

```diff
@@ -119,7 +119,11 @@
         ],
         "moduleNameMapper": {
             "\\.(css|less|scss|sass)$": "identity-obj-proxy",
-            "\\.(gif|ttf|eot|svg|png)$": "<rootDir>/test/__mocks__/fileMock.js"
+            "\\.(gif|ttf|eot|svg|png)$": "<rootDir>/test/__mocks__/fileMock.js",
+            "^@components(.*)$": "<rootDir>/lib/components$1",
+            "^@hooks(.*)$": "<rootDir>/lib/hooks$1",
+            "^@providers(.*)$": "<rootDir>/lib/providers$1",
+            "^@utils(.*)$": "<rootDir>/lib/utils$1"
```

Comment:
Jest test is failing. It does not read the alias import so I added this here to make jest configure it.